services/provider/server/server.go

```diff
@@ -817,13 +817,15 @@ func (s *OCSProviderServer) GetStorageClaimConfig(ctx context.Context, req *pb.S
 					}),
 				},
 				&pb.ExternalResource{
-					Name: "ceph-rbd",
+					Name: fmt.Sprintf("%s-groupsnapclass", req.StorageClaimName),
```

Comment:
@rewantsoni what is the reason for name change? can we keep it similar to what we have today for SC/VSClass etc?

Reply:
I am sending the exact name for VGSC here so that on client side we can create it. This name is inline with what we have in internal mode

Reply:
We have done something similar for VolumeReplicationClass where we are deciding the actual name on the provider side and sending it

Reply:
Resolving since Madhu cannot resolve the comments